### PR TITLE
Do not include exceptions in coverage chsdi.views.sitemaps

### PR DESCRIPTION
**Pull request by mariavkourouni**
now it is 100%

### CODE REVIEW

**Comment by loicgasser** [comment]
jenkins test

**Comment by loicgasser** [comment]
jenkins test

**Comment by mariavkourouni** [comment]
is this ok @loicgasser ?

File: chsdi/views/sitemaps.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def sitemap(request):
         'addresses': addresses
     }
     if params.content not in funcs:
-        raise HTTPNotFound('Missing function definition')
+        raise HTTPNotFound('Missing function definition')  # pragma: no cover
```
Comment (loicgasser) [inline chsdi/views/sitemaps.py]: this could be tested I would remove the no cover here

**Comment by loicgasser** [comment]
@mariavkourouni I don't agree with one of the 2 no cover otherwise it's ok. ;)

**Pull request merged**